app/src/main/java/edu/wpi/first/shuffleboard/app/components/WidgetTile.java

```diff
@@ -100,5 +80,4 @@ public void setShowWidget(boolean showWidget) {
   public void toggleShowWidget() {
```

Comment:
This method (and everything else related to the `showWidget` property) can be removed now that the tile can only contain the widget